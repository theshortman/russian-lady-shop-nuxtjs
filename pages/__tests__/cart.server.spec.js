/**
 * @jest-environment node
 */

import { renderToString } from '@vue/server-test-utils'
import CartPage from '../cart.vue'

test('renders correctly cart page', async () => {
  const str = await renderToString(CartPage)
  
  expect(str).toMatchSnapshot()
})