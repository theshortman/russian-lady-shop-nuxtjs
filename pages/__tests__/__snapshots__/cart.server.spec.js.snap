// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly cart page 1`] = `"<div data-server-rendered=\\"true\\"><h1>Корзина</h1> <p>Пока пусто!</p></div>"`;
